app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str,
+            health: int = 100,
+            hidden: bool = False) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.if_alive(self)
+
+    @classmethod
+    def if_alive(cls, beast: Animal) -> None:
+        cls.alive += [beast] if beast.health > 0 else []
```

Comment:
this method redundant here

Reply:
ok, i fixed it. but i thought, what we need validation of beast's health before addition to Animal.alive list
<img width="359" alt="Screenshot 2024-05-31 at 08 53 19" src="https://github.com/mate-academy/py-herbivores-and-carnivores/assets/105336389/ea6f2f6d-6dae-4083-8f00-bc02afd468fa">